src/chrome/content/rules/Tandlakarforbundet.se.xml

```diff
@@ -1,7 +1,3 @@
-<!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://tandlakarforbundet.se/ => https://www.tandlakarforbundet.se/: (7, 'Failed to connect to www.tandlakarforbundet.se port 443: Connection refused')
-Fetch error: http://www.tandlakarforbundet.se/ => https://www.tandlakarforbundet.se/: (7, 'Failed to connect to www.tandlakarforbundet.se port 443: Connection refused') tandlakarforbundet is a swedish labour union -->
 <ruleset name="Tandlakarforbundet.se">
```

Comment:
It doesn't really matter here but just so you know, your script sometimes removes comments that weren't added by travis